### PR TITLE
Added html for footer

### PR DESCRIPTION
**Pull request by Denis20101977**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -153,6 +153,123 @@ <h2 class="cashback__title">and cashback!</h2>
...
+            <li class="column__item">
+              <a href="mailto:hello@zen.com">hello@zen.com</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <!-- Info -->
+      <div class="footer__info">
+        <a class="footer__logo" href="htpps://example.com">
+          <img src="assets/icons/logo_2.svg" alt="">
```
Comment (Palych18) [inline index.html:261]: Delete or fill in "alt"

File: index.html [hunk 2]
```diff
@@ -153,6 +153,123 @@ <h2 class="cashback__title">and cashback!</h2>
...
+        </a>
+        <p class="footer__copy">ZEN is a licensed financial institution under the supervision of the Central Bank of Lithuania approved by European Banking Authority for 31 countries.</p>
+        <p class="footer__copy">Registration number of the company 304749651, VAT ID LT100011714916.</p>
+        <p class="footer__copy">License of electronic money institutions number LB000457.</p>
+        <p class="footer__copy">Share capital 2.185.000,00 EUR.</p>
+        <p class="footer__copy">Company address: Mėsinių g. 5, LT-01133, Vilnius, Lithuania.</p>
+        <p class="footer__copy">Sales Office: ul. Emilii Plater 53, 00-113 Warsaw, Poland.</p>
+        <a class="footer__dev" href="htpps://example.com">
+          <img src="assets/icons/developer.svg" alt="">
```
Comment (Palych18) [inline index.html:270]: Delete or fill in "alt"

**Review by Palych18 — COMMENTED** [review]
Stick to one style for img

File: index.html (hunk 1 above)
Comment (Denis20101977) [reply index.html]: Deleted "alt"

File: index.html (hunk 2 above)
Comment (Denis20101977) [reply index.html]: Deleted "alt"

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**